### PR TITLE
various changes

### PR DESCRIPTION
**Pull request by DavidBJaffe**
- track and report barcode fate
- fix compilation caching by GitHub Actions
- add parseable field vj_seq_nl, the DNA sequence of V..J, but starting after the leader
- add parseable field vj_aa_nl, the amino acid sequence of V..J, but starting after the leader
- add chain variable aa% = percent identity with donor reference, outside junction region.
- similarly, add dna%
- fix bug: CONST_IGH did not work with META
- slight clarification to language on heuristics page

### CODE REVIEW

File: enclone/src/explore.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ use string_utils::*;
 use vector_utils::*;
 
 // Exploratory code, in which we examine the data, so as to help decide on the
-// best definition of exact subclonotype.  Antiquated.
+// best definition of exact subclonotype.  Super antiquated.
```
Comment (wyattmcdonnell) [inline enclone/src/explore.rs:10]:
```suggestion
// best definition of exact subclonotype.  Fossilized.
```

File: enclone/src/misc1.rs [hunk 2]
```diff
@@ -190,10 +191,14 @@ pub fn lookup_heavy_chain_reuse(
...
+    fate: &mut Vec<HashMap<String, String>>,
+) {
     if !ctl.clono_filt_opt.ncross {
-        // Get the list of dataset origins.  Here we allow the same origin name to have been used for
-        // more than one donor, as we haven't explicitly prohibited that.
+        // Get the list of dataset origins.  Here we allow the same origin name to have been used
+        // for more than one donor, as we haven't explicitly prohibited that.
 
         let mut origins = Vec::<(String, String)>::new();
         for i in 0..ctl.origin_info.n() {
```
Comment (wyattmcdonnell) [inline enclone/src/misc1.rs:204]: This reminds me... isn't this explicitly prohibited in `cellranger vdj aggr`?

Comment (DavidBJaffe) [reply enclone/src/misc1.rs:204]: Not sure!

Comment (wyattmcdonnell) [reply enclone/src/misc1.rs:204]: @niranjan628496 and @sreenathkrishnan bringing this to your attention... we may wish to look at this again in the future

**Comment by DavidBJaffe** [comment]
jenkins retest this please

**Comment by wyattmcdonnell** [comment]
Jenkins retest this please

**Comment by meryllewis** [comment]
jenkins retest this please

**Comment by meryllewis** [comment]
jenkins retest this please

File: enclone/src/misc1.rs (hunk 2 above)
Comment (niranjan628496) [reply enclone/src/misc1.rs:204]: yes, this is prohibited in aggr.
I think we may want to enforce that origin name is a unique identifier.

**Pull request merged**